streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStore.java

```diff
@@ -79,6 +79,13 @@ public void put(K key, V value) {
     }
 
     @Override
+    public void putIfAbsent(K key, V value) {
+        if (get(key) == null) {
+            put(key, value);
+        }
+    }
```

Comment:
I think it would be better to invoke `inner.putIfAbsent` so that we can benefit from a more efficient implementation where available.